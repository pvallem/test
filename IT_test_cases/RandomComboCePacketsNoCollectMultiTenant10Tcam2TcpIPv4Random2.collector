bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\n\013\014\r\000\000"
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\002"
    dst_address: "\n\013\014\r\000\002"
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000040
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\n\013\014\r\000\001"
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\204\001L"
    dst_address: "\314^9\344"
    src_port: 46770
    dst_port: 51780
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "G\357`\342"
    dst_address: "J^k\357"
    src_port: 34370
    dst_port: 34370
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
