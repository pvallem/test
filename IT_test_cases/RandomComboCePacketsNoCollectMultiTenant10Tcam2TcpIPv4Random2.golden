flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\n\013\014\r\000\000"
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000000
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\314^9\344"
    dst_address: "=\204\001L"
    src_port: 51780
    dst_port: 46770
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\n\013\014\r\000\001"
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000020
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "G\357`\342"
    dst_address: "J^k\357"
    src_port: 34370
    dst_port: 34370
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\002\002\002\002\002\002"
    dst_address: "\n\013\014\r\000\002"
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000040
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
